packages/create-workleap/package.json

```diff
@@ -2,9 +2,9 @@
   "name": "create-workleap",
   "version": "0.0.0",
   "description": "Foundry-CLI Workleap project starter kit",
-  "bin": "./dist/bin/bin.js",
-  "main": "./dist/create-workleap.js",
-  "types": "./dist/create-workleap.d.ts",
+  "bin": "./dist/index.js",
+  "main": "./dist/index.js",
```

Comment:
You don't need a `main` for this one as it will only be used as an executable.

Reply:
Do I remove the `types` too?